jit/v2/JIT_v2.0.0.yaml

```diff
@@ -940,6 +936,263 @@ components:
         example: '2'
       description: |
         An API-Version header **MAY** be added to the request (optional); if added it **MUST** only contain **MAJOR** version. API-Version header **MUST** be aligned with the URI version.
+    ##############
+    # Query params
+    ##############
+
+    # Technical query parameters
+
+    portCallIDQueryParam:
+      in: query
+      name: portCallID
+      description: |
+        The **Port Call Identifier** to filter by. Specifying this filter will only return **Port Call Service** objects including the `portCallID`.
```

Comment:
Does it mean here that, if a property is present in the filter it should be given back in the response even if it's optional? Are we going with that?